### ISSUE TITLE
[1.12.2-1.16.5] Citizens despawn after server reboot

### ISSUE THREAD

**Issue by VOrlando520** [Luohuayu/CatServer#624]
Latest Catserver 1.16.5

After a restart, any created Citizens npc despawn and the plugin has to be reloaded for them to appear again.
No errors in console or logs. Seems like the patcher was never carried over from 1.12 to 1.16.5.

Wondering if that is possible. It is a widely used plugin that has a variety of functions and plugs into other useful plugins.

**Comment by XionioXMaster** [comment]
Why not compiled on the topic? Are there mods? Tried to play on server without mods

**Comment by Kotori0629** [comment]
> 

No, this problem was discovered a long time ago. This is a real problem.

**Comment by Kotori0629** [comment]
> Why not compiled on the topic? Are there mods? Tried to play on server without mods

No, this problem was discovered a long time ago. This is a real problem.

**Comment by VOrlando520** [comment]
> Why not compiled on the topic? Are there mods? Tried to play on server without mods

Yeah Ive tried mods and no mods on a clean test server and still had the same issue.

**Comment by VOrlando520** [comment]
This ever get looked into? Its been 2 months
